doc/config-v2.rst

```diff
@@ -195,7 +195,7 @@ Default settings related to environments names/variants
 tox comes with predefined settings for certain variants, namely:
 
 * ``{easy,pip}`` use easy_install or pip respectively
-* ``{py24,py25,py26,py27,py31,py32,py33,py34,pypy19]`` use the respective
+* ``{py27,py36,py35,py34,pypy19]`` use the respective
```

Comment:
nit: I'd expect these to be in increasing order instead of decreasing

Reply:
same for the rest of the docs -- we should consistently:
- mention pythonx.x, pythony.y in increasing order
- list tox environments in increasing order

I understand this makes your diff less minimal, but it makes the docs read better :)